packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListListbox.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+
+import InfiniteList, { Item } from '../../../index';
+import styles from './InfiniteListTestCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const InfiniteListListbox = () => (
+  <InfiniteList
+    id="test-infinite-list"
+    isFinishedLoading
+    role="listbox"
+    ariaLabel="Listbox"
```

Comment:
If this is fixing accessibility, should all infinite list examples be updated to use it? plus this was made a required prop; are the other examples throwing console errors for missing required prop?

Reply:
Yes, since `ariaLabel` is a required prop, the other examples were throwing console errors.
I have addressed that here : https://github.com/cerner/terra-framework/pull/999/commits/a1ebbff90d1499551ed4e6a5432370509814edaf